### ISSUE TITLE
👑[Enhancement] Revert to or Enable old family/liberary share

### ISSUE THREAD

**Issue by F00lsGold** [BeyondDimension/SteamTools#3497]
Hi, I've been a multi-account user for a long time, back when Steam had some serious regional price discrimination.
Recently, I accidentally created a NEW steam family, it fxxked up all my sharing. Now I have to login to different account to play games.
The new family share doesn't function in my case, as my other accounts are in a different region.
Steam says the new one **completely replace** the old one, but it seems to me they exist at the same time, I just need a way to toggle it back to the older version. Here's my discovery.

09/26/2024

I have main account A, alt account B, C.
I enabled NEW family sharing on A, lost access to lib B, C.
I thought I could just use the new family function, hence enabled new sharing on B.
Steam tells me since they are in a different region, B cannot join A's family.
I deleted A's family, re-apply for B's lib, here's what I got.
![image](https://github.com/user-attachments/assets/e2fc2cbb-9509-437f-a310-83702a2ba2a0)

I then re-apply for C's lib, notice C didn't have new family function enabled. It succeed.
![image](https://github.com/user-attachments/assets/4221a902-086d-41c0-a93b-d0544cc22e5c)

A and B's family page looks like this
![image](https://github.com/user-attachments/assets/1b01e1d6-b686-4b17-9184-bb6776960da9)

While C's looks like this
![image](https://github.com/user-attachments/assets/e087b0e6-0844-490d-86f4-2ad29f8975c0)

It seems to me that they've just disabled a layout, is there a way to enable this layout?

**Comment by rmbadmin** [comment]
`
win+r：steam://open/console 
`
`
steam console: device_authorize_local [description]
`

For now, you can manually use the old version family sharing in this way, but the prerequisite is that it cannot be used simultaneously with the new family sharing

**Comment by jayfolder** [comment]
What would you have to put in the `[description]` part of the command?

**Comment by F00lsGold** [comment]
> `win+r：steam://open/console ` `steam console: device_authorize_local [description]`
> 
> For now, you can manually use the old version family sharing in this way, but the prerequisite is that it cannot be used simultaneously with the new family sharing

Should I be doing _device_authorize_local_ on account B, [description] is account name A?

**Comment by F00lsGold** [comment]
@rmbadmin 
![image](https://github.com/user-attachments/assets/a050edb4-b1c5-4dd1-8d72-3c7c53f4594a)
Like this?

**Comment by RadityaDito** [comment]
> `win+r：steam://open/console ` `steam console: device_authorize_local [description]`
> 
> For now, you can manually use the old version family sharing in this way, but the prerequisite is that it cannot be used simultaneously with the new family sharing

Is it work? can you give us full tutorial for it please

**Comment by kotovasia5120** [comment]
@F00lsGold yore god )

**Comment by F00lsGold** [comment]
@kotovasia5120 Does it work for you, cuz it surely doesn't work for me

**Comment by kotovasia5120** [comment]
> @kotovasia5120 Does it work for you, cuz it surely doesn't work for me

I reinstalled steam and my old menu stopped showing. I have not activated a new family.After the command was updated, the old menu appeared and I was able to use other libraries. This day greenluma was working, but the next day it stopped. But access to other libraries works. Writes that it will be disabled in 2025.

**Comment by kohega** [comment]
I just did `device_authorize_local [account public name]` and it seems to have worked

**Comment by F00lsGold** [comment]
@Kohega 
Would you post a screenshot or sort?

I did `device_authorize_local [account public name]` on my alt acc, where name is my main, I did not work

**Comment by kohega** [comment]
@F00lsGold
![Copie d'écran_20241031_191236](https://github.com/user-attachments/assets/0952e975-839c-4d54-899a-a92c97b125bd)
![Copie d'écran_20241031_191457-1](https://github.com/user-attachments/assets/31988a60-47ad-428e-9564-7f831d174833)
I just installed Steam, added my 2nd account, launch `steam steam://open/console` and ran `device_authorize_local [Kohega]` in my case

**Comment by kotovasia5120** [comment]
> @Kohega Would you post a screenshot or sort?
> 
> I did `device_authorize_local [account public name]` on my alt acc, where name is my main, I did not work

It seems that you need to write the computer name obtained using device_authorize_status there.
Descripcion item.

**Comment by Yoda381** [comment]
@kotovasia5120 so i have to go on my account B where i want the account to be authorised and enter the account A name(the computer name) and then will i be able to get the account A library shared with account B

**Comment by Yoda381** [comment]
Plz help me out this new library sharing feature is really bad

**Comment by kohega** [comment]
No, I did it from the alt acc that I wanted to share to my main account

**Comment by Yoda381** [comment]
@Kohega so u went to alt account nd enabled sharing there nd shared ur alt account games to ur main account?

**Comment by RadityaDito** [comment]
Is it still work? because after the new update mine not work

**Comment by gherbetto** [comment]
![Image](https://github.com/user-attachments/assets/e473829e-2f1f-47c4-b225-d33dd87097c2)

**Comment by kotovasia5120** [comment]
> ![Image](https://github.com/user-attachments/assets/e473829e-2f1f-47c4-b225-d33dd87097c2)

The old version was completely removed.
